src/shared/index.tsx

```diff
@@ -6,5 +6,6 @@ export const URL = {
 export const apiUrl = () => ({
   products: {
     getAll: "/api/products",
+    add: "/api/products/create",
```

Comment:
The api path is `create` therefore the key should be `create` as well.
```suggestion
    create: "/api/products/create",
```

Reply:
Fixed